src/modules/esl-panel/core/esl-panel-group.ts

```diff
@@ -3,26 +3,38 @@ import {attr, ESLBaseElement} from '../../esl-base-element/core';
 import {afterNextRender} from '../../esl-utils/async/raf';
 import {bind} from '../../esl-utils/decorators/bind';
 import {CSSUtil} from '../../esl-utils/dom/styles';
-import {ESLMediaQuery} from '../../esl-media-query/core';
-
+import {ESLMediaRuleList} from '../../esl-media-query/core';
 import {ESLPanel} from './esl-panel';
 
-@ExportNs('PanelStack')
-export class ESLPanelStack extends ESLBaseElement {
-  public static is = 'esl-panel-stack';
+@ExportNs('PanelGroup')
+export class ESLPanelGroup extends ESLBaseElement {
+  public static is = 'esl-panel-group';
 
-  @attr() public accordionTransformation: string;
+  @attr() public mode: string;
   @attr({defaultValue: 'animate'}) public animationClass: string;
   @attr({defaultValue: 'accordion'}) public accordionClass: string;
   @attr({defaultValue: 'auto'}) public fallbackDuration: number;
 
+
+  private _currentMode: string | null;
```

Comment:
Do we need a separate _currentMode field?
Can we just use ESLMediaRuleList state? (If remember correct it is already cached and have "active" accessors for rule and value)

Reply:
Done